### DISCUSSION TITLE
No se está cargando de manera correcta los campos nombres y code de una sm

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#490] [4. Bug Reports] [answered]
Hola

Adjunto el video de evidencia

https://drive.google.com/file/d/1uNXBZ0B5QXzht0JKofIb2sKngeasyZhh/view

Gracias =>

**Comment by jdurandpaz** [comment, marked as answer]
se crea issue https://github.com/Cotalker/documentation/issues/492
